http.go

```diff
@@ -0,0 +1,117 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tspclient
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+// maxBodyLength specifies the max content can be received from the remote
+// server.
+// The legnth of a regular TSA response with certificates is usually less than
+// 10 KiB.
+var maxBodyLength = 1 * 1024 * 1024 // 1 MiB
+
+// TimestampQuery is the content-type of timestamp query.
+// RFC 3161 3.4
+const TimestampQuery = "application/timestamp-query"
```

Comment:
Because user may want to implement their own `Timestamper` defined in `tspclient.go` (the alternative is to use the one provided by this library in `http.go`). If they choose to do so, they will need this variable as defined in RFC 3161.